csv2sql.drush.inc

```diff
@@ -233,4 +243,45 @@ function csv2sql_get_column_name($col_name) {
   return '_' . trim(strtolower(str_replace(array('-', ' ', '/'), '_', $col_name)));
 }
 
+/**
+ * Scans the directory given by the user in the command,
+ * Scans the sub-directories as well,
+ * Searches for files with 'csv' extension to be added to the convert process.
+ *
+ * @param $dir
+ *  The path of the directory provided in the command.
+ *
+ * @param $files
+ *  An array containing a name of file if the user added
+ *  a file path in addition to the directory path.
+ *
+ * @return array
+ *  An array of 'csv' files found in the directory.
+ */
+function scan_directory($dir, $files) {
+  // If the directory does not exist, print a message, stop the function
+  // and return the file path we got from the command line (If there's any).
+  if (!file_exists($dir)) {
+    drush_print('Given directory does not exist.');
+    return $files;
+  }
+  // Save the array of files in a global variable.
+  $csv_files = variable_get('csv_files') ? variable_get('csv_files') : $files;
+  // Add a trailing slash and a * so the glob returns everything in the directory,
+  // Trim trailing slash and add them here, To make sure we never have '//'.
+  $items_in_dir = glob(rtrim($dir, '/') . DIRECTORY_SEPARATOR . '*');
+
+  foreach ($items_in_dir as $item) {
+    if (pathinfo($item, PATHINFO_EXTENSION) == 'csv') {
+      $csv_files[] = $item;
+      variable_set('csv_files', $csv_files);
+    }
+    if(is_dir($item)) {
+      scan_directory($item, $csv_files);
+    }
+  }
+
+  return variable_get('csv_files');
```

Comment:
this should be documented in the README. maybe var name should be ``csv2sql_files_path``?